src/ensembl/webpack/webpack.config.dev.js

```diff
@@ -67,7 +67,9 @@ const devConfig = {
     // rules to proxy requests to the backend server in development
     proxy: {
       '/api': {
-        target: 'https://staging-2020.ensembl.org',
+        // IMPORTANT! NOTE: change this to the original url once the PR is approved
+        target: 'http://193.62.55.91:30667',
+        // target: 'https://staging-2020.ensembl.org',
```

Comment:
NOTE TO SELF: Change this to the original `target` URL once the PR has been approved